roles/matrix-cactus-comments/defaults/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Cactus Comments is a federated comment system built on Matrix
+
+matrix_cactus_comments_enabled: true
+matrix_cactus_comments_container_image_self_build: false
+matrix_cactus_comments_docker_repo: "https://gitlab.com/cactus-comments/cactus-appservice.git"
```

Comment:
Just FYI, this is the git repo. The Docker Hub is here: https://hub.docker.com/r/cactuscomments/cactus-appservice

Not sure what's intended.

Reply:
This is intended (I think) as this is the repo where the Dockerfile for self-building can be found AFAIK